### PR TITLE
Travis CI

### PR DESCRIPTION
**Pull request by cronburg**
See commit message for details.

This should be merged after #13 and #17.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+timing.temp
+dist
+/.stack-work
```
Comment (dnunez02) [inline .gitignore]: This is meant to be .stack-work?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+/stack.yaml
```
Comment (dnunez02) [inline .gitignore]: Ditto.

File: README.md [hunk 3]
```diff
@@ -5,28 +5,47 @@ A tool for inferring strictness annotations in Haskell programs using a genetic
...
 > stack setup
 > stack build
+> stack install
 ```
 
 For newer versions of GHC, `stack init` may fail to resolve the necessary packages,
 in which case you will need to initialize with `stack init --solver --install-ghc`.
 
-Then run the executable with
+WARNING: Autobahn will overwrite the original source of your project. You should
```
Comment (dnunez02) [inline README.md:22]: We should fix this instead of leaving a WARNING.

File: src/Main.hs [hunk 4]
```diff
@@ -34,25 +35,26 @@ checkBaseProgram baseTime baseMetric = if baseTime == -1
                                                  "Autobahn cannot optimize."
                                             else putStr $ "Base measurement is: " ++ (show baseTime)
 
-fitness :: FilePath -> String -> Double -> Int64 -> MetricType -> [FilePath] -> [BangVec] -> IO Time
-fitness projDir args timeLimit reps metric files bangVecs = do
+--fitness :: FilePath -> String -> Double -> Int64 -> MetricType -> [FilePath] -> [BangVec] -> IO Time
+--fitness projDir args timeLimit reps metric files bangVecs = do
+fitness cfg reps files bangVecs = do
```
Comment (dnunez02) [inline src/Main.hs:40]: Good call replacing most of the args with Cfg.

Add the type of this function.

**Review by dnunez02 — CHANGES_REQUESTED** [review]
Needs another once over for small things. But it looks good.

File: README.md (hunk 3 above)
Comment (cronburg) [reply README.md:22]: Making an issue now.

**Pull request merged**